### ISSUE TITLE
Wikipedia

### ISSUE THREAD

**Issue by the-moog** [BlissRoms-x86/Documentation#15]
I noticed that Wikipedia lacked Bliss on the page "List of Linux based distributions" so I added it https://en.wikipedia.org/wiki/List_of_Linux_distributions#Android-based.  You may want to review/edit and/or create pages for the various broken links that relate to the BlissOS project.

Jay.

**Comment by electrikjesus** [comment]
We will have to wait for the addition to be public. Thank you

**Comment by electrikjesus** [comment]
Hey Jay, my IP address does not allow me to edit your proposed updates. One issue I see is the Waydroid description and mention. Waydroid is not based on Bliss OS, it is based on LineageOS.

**Comment by Xtr126** [comment]
I personally think that Waydroid should be not included in a list of Linux distributions, it may cause confusion. Waydroid is a project that allows you to run Android applications on your Linux desktop, not a Linux distribution on it's own.

**Comment by electrikjesus** [comment]
I agree with @Xtr126 on Waydroid not being an Android/Linux distro. 

As for the other missing links to the Bliss OS/Bliss ROM wiki pages, all previous drafts have been denied, and there has never been any feedback as to why, other than "no advertising", which there was not any in the project description. It was a history of the project. So there is some sketchy reviewing policies in place there.

**Comment by realyukii** [comment]
I can't create new issue so I will comment here, the link mentioned in [this page](https://docs.blissos.org/installation/advanced-installation/) isn't working (404):
> This is only for advanced users. For regular users, please visit our main [Installation Guide found here.](https://docs.blissos.org/installation/advanced-installation/installation-guide.md)

and the section "Documentation issue" from template issue on [this github repo](https://github.com/BlissRoms-x86/support/issues/new/choose)  point to the [archived version](https://github.com/BlissRoms-x86/docs)
